sdk/textanalytics/Azure.AI.TextAnalytics/src/MultiLabelClassifyAction.cs

```diff
@@ -10,15 +10,15 @@ namespace Azure.AI.TextAnalytics
     /// For example, the target project and deployment names are required
     /// for a successful custom classification action.
     /// </summary>
-    public class MultiCategoryClassifyAction
+    public class MultiLabelClassifyAction
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="MultiCategoryClassifyAction"/>
+        /// Initializes a new instance of the <see cref="MultiLabelClassifyAction"/>
         /// class which allows callers to specify details about how to execute
         /// a Multi Category Classification action on a set of documents.
```

Comment:
```suggestion
        /// a Multi Label Classification action on a set of documents.
```

Reply:
Good catch. I didn't think about all the written examples when doing refactoring. I'll go through and find them all.